src/components/MessagesList/MessagesGroup/Message/MessagePart/PollVotersDetails.vue

```diff
@@ -24,7 +24,7 @@
 		<button slot="trigger"
 			tabindex="0"
 			class="poll-voters-details">
-			<AvatarWrapperSmall v-for="(item, index) in details"
+			<AvatarWrapperSmall v-for="(item, index) in details.slice(0, 8)"
```

Comment:
Should we have a guest avatar with `…` in case there where more than the slice?

Reply:
yes we should, separate issue tho